Fails to build with Sphinx 3.4.3

Hello,

On Fedora we are testing the dependants of Sphinx whenever we update to a new version, and it seems that sphinx-notfound-page fails to build with this error message:

```
================================== FAILURES ===================================
__________________________ test_sphinx_resource_urls ___________________________

app = <SphinxTestApp buildername='html'>
status = <_io.StringIO object at 0x7fbf98785280>
warning = <_io.StringIO object at 0x7fbf98785700>

    @pytest.mark.sphinx(srcdir=srcdir)
    def test_sphinx_resource_urls(app, status, warning):
        app.build()
        path = app.outdir / '404.html'
        assert path.exists()
    
        content = open(path).read()
    
        chunks = [
            # Sphinx's resources URLs
            '<script type="text/javascript" src="/en/latest/_static/jquery.js"></script>',
            '<script type="text/javascript" src="/en/latest/_static/underscore.js"></script>',
            '<script type="text/javascript" src="/en/latest/_static/doctools.js"></script>',
        ]
    
        if sphinx.version_info >= (1, 8):
            chunks.append(
                '<script type="text/javascript" src="/en/latest/_static/language_data.js"></script>',
            )
    
        for chunk in chunks:
>           assert chunk in content
E           assert '<script type="text/javascript" src="/en/latest/_static/jquery.js"></script>' in '\n<!DOCTYPE html>\n\n<html>\n  <head>\n    <meta charset="utf-8" />\n    <meta name="viewport" content="width=device-..."https://github.com/bitprophet/alabaster">Alabaster 0.7.12</a>\n      \n    </div>\n\n    \n\n    \n  </body>\n</html>'

tests/test_urls.py:456: AssertionError
```

--- a/tests/test_urls.py
+++ b/tests/test_urls.py
@@ -33,7 +33,7 @@ def test_parallel_build():
     # TODO: migrate to `app.build(..., parallel=2)` after merging
     # https://github.com/sphinx-doc/sphinx/pull/8257
     subprocess.check_call('sphinx-build -j 2 -W -b html tests/examples/parallel-build build', shell=True)
-    
+
 @pytest.mark.sphinx(srcdir=srcdir)
 def test_404_page_created(app, status, warning):
     app.build()
@@ -529,7 +529,7 @@ def test_sphinx_resource_urls(app, status, warning):
             '<script src="/en/latest/_static/doctools.js"></script>',
         ]
 
-    if sphinx.version_info >= (1, 8):
+    if (1, 8) <= sphinx.version_info < (3, 4, 0):
         if sphinx.version_info < (2, 4, 0):
             chunks.append(
                 '<script type="text/javascript" src="/en/latest/_static/language_data.js"></script>',

--- a/tests/test_urls.py
+++ b/tests/test_urls.py
@@ -529,6 +529,9 @@ def test_sphinx_resource_urls(app, status, warning):
             '<script src="/en/latest/_static/doctools.js"></script>',
         ]
 
+    # This file was added to all the HTML pages in Sphinx>=1.8. However, it was
+    # only required for search page. Sphinx>=3.4 fixes this and only adds it on
+    # search. See (https://github.com/sphinx-doc/sphinx/blob/v3.4.0/CHANGES#L87)
     if (1, 8) <= sphinx.version_info < (3, 4, 0):
         if sphinx.version_info < (2, 4, 0):
             chunks.append(

--- a/tox.ini
+++ b/tox.ini
@@ -2,7 +2,7 @@
 envlist =
   docs
   py27-sphinx{17,18}
-  py{36,37,38}-sphinx{17,18,20,21,22,23,24,30,31,32,33}
+  py{36,37,38}-sphinx{17,18,20,21,22,23,24,30,31,32,33,34}
 
 [testenv]
 deps =
@@ -20,6 +20,7 @@ deps =
   sphinx31: sphinx~=3.1.0
   sphinx32: sphinx~=3.2.0
   sphinx33: sphinx~=3.3.0
+  sphinx34: sphinx~=3.4.0
 commands = pytest {posargs}
 
 [testenv:py38-sphinx30]

